flocker/control/test/configurations/configuration_1_v5.json

```diff
@@ -0,0 +1 @@
+{"deployment": {"persistent_state": {"blockdevice_ownership": {"values": [], "$__class__$": "PMap"}, "$__class__$": "PersistentState"}, "nodes": [{"applications": {"values": [["myapp", {"volume": {"mountpoint": {"path": "/xxx/yyy", "$__class__$": "FilePath"}, "manifestation": {"dataset": {"deleted": false, "dataset_id": "4e7e3241-0ec3-4df6-9e7c-3f7e75e08855", "metadata": {"values": [["name", "myapp"]], "$__class__$": "PMap"}, "maximum_size": null, "$__class__$": "Dataset"}, "primary": true, "$__class__$": "Manifestation"}, "$__class__$": "AttachedVolume"}, "memory_limit": null, "name": "myapp", "links": [], "$__class__$": "Application", "restart_policy": {"$__class__$": "RestartNever"}, "image": {"tag": "7.6", "repository": "postgresql", "$__class__$": "DockerImage"}, "command_line": null, "environment": {"values": [], "$__class__$": "PMap"}, "running": true, "ports": [], "cpu_shares": null}]], "$__class__$": "PMap"}, "manifestations": {"values": [["4e7e3241-0ec3-4df6-9e7c-3f7e75e08855", {"dataset": {"deleted": false, "dataset_id": "4e7e3241-0ec3-4df6-9e7c-3f7e75e08855", "metadata": {"values": [["name", "myapp"]], "$__class__$": "PMap"}, "maximum_size": null, "$__class__$": "Dataset"}, "primary": true, "$__class__$": "Manifestation"}]], "$__class__$": "PMap"}, "uuid": {"hex": "ab294ce4-a6c3-40cb-a0a2-484a1f09521c", "$__class__$": "UUID"}, "$__class__$": "Node"}], "leases": {"values": [], "$__class__$": "PMap"}, "$__class__$": "Deployment"}, "version": 5, "$__class__$": "Configuration"}
```

Comment:
Acknowledged.

This is generated and read by a script though:

`flocker/control/test/configurations/configuration_v5.py`

So I guess not tempted to add it manually right now? Not trying to be difficult, just trying to both:

A) have the code actually and truly generation these files
B) Not have random code that just adds a newline added to the generation script (And I think it would also have to be added to the verification test that I added earlier).